src/main/java/com/blamejared/compat/tconstruct/actions/SetAccuracyAction.java

```diff
@@ -0,0 +1,64 @@
+package com.blamejared.compat.tconstruct.actions;
+
+import com.blamejared.brackets.util.IMaterial;
+import minetweaker.IUndoableAction;
+import slimeknights.tconstruct.library.materials.FletchingMaterialStats;
+import slimeknights.tconstruct.library.materials.IMaterialStats;
+import slimeknights.tconstruct.library.materials.Material;
+
+/**
+ * Created by Rinart73 on 7/25/2017.
+ */
+public class SetAccuracyAction implements IUndoableAction {
+
+    private final IMaterial material;
+    private final String stat;
+    private final float newValue;
+    private final float oldValue;
+
+    public SetAccuracyAction(IMaterial material, String stat, float newValue) {
+        this.material = material;
+        this.stat = stat;
+        this.newValue = newValue;
+        this.oldValue = ((FletchingMaterialStats) ((Material) material.getInternal()).getStats(stat)).accuracy;
+    }
+
+    private static void set(Material material, String stat, float accuracy) {
```

Comment:
Because it was like this initially and I didn't change it. I'm not seeing any reason to remove 'static' besides the possibility that someone will want to override it. Okay, I'll make this just private.